libr/debug/p/native/linux/linux_debug.c

```diff
@@ -97,6 +97,68 @@ int linux_handle_signals(RDebug *dbg, int tid) {
 		// siginfo.si_code -> HWBKPT, USER, KERNEL or WHAT
 		// TODO: DO MORE RDEBUGREASON HERE
 		switch (dbg->reason.signum) {
+		case SIGABRT: // 6 / SIGIOT // SIGABRT
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGALRM:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGBUS:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;
+			break;
+		case SIGCHLD:
+			dbg->reason.type = R_DEBUG_REASON_SIGNAL;;
+			break;
+		case SIGCONT:
```

Comment:
no need to repeat the same code over and over just use multiple cases in a single statement, but if you do what i explained in the previous comment it shouldnt be needed to simplify this